### ISSUE TITLE
AF::Base initialization should not require a FCR4 connection

### ISSUE THREAD

**Issue by barmintor** [samvera/active_fedora#1060]
You should be able to initialize a new AF::Base, sans id, without an open connection to Fedora. This would greatly facilitate unit testing model behaviors (e.g. Rspec.stub_model). Complexity in testing makes it difficult for me to establish the testing and code quality standards I associate with the Hydra community in my own team's work.

**Comment by barmintor** [comment]
Reassigning @mjgiarlo until it is assignable to a person who will do it.

**Comment by mjgiarlo** [comment]
@barmintor My fear is that assigning it to me will send the signal that someone's already working on it, so no one will pick it up. Perhaps this should be a topic for a Dev Congress?

**Comment by atz** [comment]
@mjgiarlo Very little is getting picked up by anyone, so I'm not sure that is much of a risk.

**Comment by atz** [comment]
@barmintor Since `Base` is just a list of extends/includes, where does the requirement actually come from?

The amount of autoloading `ActiveFedora` does makes it hard to determine, but the amount of time `require "active_fedora"` takes suggests it might not be a problem with Base so much as `lib/active_fedora.rb` itself.

But in IRB, without fedora running, I can do:
```
> require 'active_fedora'
 => true 
> x = ActiveFedora::Base.new
 => #<ActiveFedora::Base id: nil> 
> x.persisted?
 => false 
> x.save
Faraday::ConnectionFailed: Failed to open TCP connection to 127.0.0.1:8984 (Connection refused - connect(2) for "127.0.0.1" port 8984)
```

Isn't this what you wanted?

**Comment by jcoyne** [comment]
Fixed in 6a2b1cd63d59ddf156cd9cdbc408f324ccbe7b6d ?

**Comment by barmintor** [comment]
@jcoyne probably, it looks like?
